Example Code/Reflowduino_Demo/Reflowduino_Demo.ino

```diff
@@ -108,6 +119,8 @@ Adafruit_MAX31855 thermocouple(MAX_CS);
 #define stopChar '!' // App is receiving command to stop reflow process (process finished!)
 #define startReflow 'A' // Command from app to "activate" reflow process
 #define stopReflow 'S' // Command from app to "stop" reflow process at any time
+enum STATE{idle,preheat,soak,reflowA,reflowB,cool};
```

Comment:
Traditional reflow profiles don't have two separate reflow zones. Was there a specific reason for this?

Reply:
[see this datasheet](https://www.sra-shops.com/docs/srasolder/datasheet/superior-solder-paste-8013-85.pdf), note stages 1 and 2 of the reflow.

Reply:
Hmmm, interesting. Maybe I was just looking at the overall "Reflow" zone in most pictures.